website/docs/advanced-topics/children.mdx

```diff
@@ -90,6 +102,53 @@ impl Component for List {
 }
 ```
 
+## Nested Children with Props
+
+Nested component properties can be accessed and mutated if the containing component types its children. In the following example, the `List` component can wrap `ListItem` components. For a real-world example of this pattern, check out the `yew-router` source code. For a more advanced example, check out the `nested-list` example in the main yew repository.
```

Comment:
> For a real-world example of this pattern, check out the `yew-router` source code. For a more advanced example, check out the `nested-list` example in the main yew repository.
This should be under the [Further Reading section](https://yew.rs/docs/next/concepts/function-components/children#further-reading)